src/lib/server.ts

```diff
@@ -233,6 +238,36 @@ export class ClientsController {
       this.eventForwarder = new EventForwarder(this);
       this.eventForwarder.start();
     }
+
+    if (this.loggingEventForwarder === undefined) {
+      this.loggingEventForwarder = new LoggingEventForwarder(this, this.driver);
+    }
+  }
+
+  get loggingEventForwarderStarted(): boolean {
+    if (
+      !this.loggingEventForwarder ||
+      this.loggingEventForwarder === undefined
+    ) {
+      return false;
+    }
+    return this.loggingEventForwarder.started;
```

Comment:
This should work too:
```suggestion
    return this.loggingEventForwarder?.started === true;
```